oat-sa/tao-core
# [ACTP-358] Accessibility issues on Login page

Related to : https://oat-sa.atlassian.net/browse/ACTP-358
 
- Change element with id="login box" to use &lt;main&gt; tag
- Update package.json
  
#### How to test
 
- prepare an instance of TAO with taoAct extension
- navigate to login page
- check element with id="login box" uses &lt;main&gt; tag
  
#### Dependencies
   
Requires :
 - [ ] https://github.com/oat-sa/tao-core-ui-fe/pull/103
